packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/webview_cookie.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Configuration to use when creating a new [WebViewPlatformController].
+///
+/// The `autoMediaPlaybackPolicy` parameter must not be null.
+class WebViewCookie {
+  /// Construct a new [WebViewCookie].
```

Comment:
nit: Construct -> Constructs (present tense is now recommended for consistency)
```suggestion
  /// Constructs a new [WebViewCookie].
```